src/view/components/checkboxList.jsx

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import Checkbox from "@react/react-spectrum/Checkbox";
+
+import "./checkboxList.styl";
+
+class CheckboxList extends React.Component {
+  onChange = (isChecked, event) => {
+    const checkboxValue = event.nativeEvent.target.value;
+    const { onChange } = this.props;
+    let { value } = this.props;
+
+    if (!value) {
+      value = [];
```

Comment:
is it okay to remove this check by defaulting value to `[]` while creating.
`let { value = []} = this.props;`

Reply:
Yes, I can! Sorry, this bit of code was copied from es5 and I didn't do a good enough job converting it to es6. Done!